packages/icons/stories/icon.all.ts

```diff
@@ -543,3 +543,198 @@ export const allIcons = [
   ZipIcon,
   ZipSolidIcon,
 ];
+export const allIconsNames = [
```

Comment:
Just noticed that this list is missing a few e.g. DarkIcon. Maybe we can update this file to make it a bit easier to manage e.g. not needing to be updated if we generate different icons in the future...

something like:
```tsx
// icon.all.ts
import * as allIconsExport from "../src/components";

export const allIcons = Object.values(allIconsExport);

export const allIconsNames = Object.entries(allIconsExport).map(
  ([name, icon]) => ({
    name,
    icon,
  })
);

```